modules/dxp/apps/portal-workflow/portal-workflow-metrics-rest-impl/src/main/java/com/liferay/portal/workflow/metrics/rest/internal/resource/v1_0/InstanceResourceImpl.java

```diff
@@ -147,18 +147,26 @@ public Instance getProcessInstance(Long processId, Long instanceId)
 		overdueFilterAggregation.addChildAggregation(
 			_resourceHelper.createOverdueScriptedMetricAggregation());
 
-		TermsAggregation slaDefinitionIdTermsAggregation = _aggregations.terms(
-			"slaDefinitionId", "slaDefinitionId");
+		BooleanQuery slaInstanceResultBooleanQuery = _queries.booleanQuery();
```

Comment:
Hey @MarinhoFeliphe, please, inline the `slaInstanceResultBooleanQuery.addMustQueryClauses(...)` into the `_aggregations.filter("slaInstanceResult", ...)` one.